msprime/mutations.py

```diff
@@ -164,13 +170,17 @@ class InfiniteAllelesMutationModel(
     _msprime.InfiniteAllelesMutationModel, MutationModel
 ):
     """
-    This mutation model assigns a new allele at every mutation. To do this it uses
-    a numerical counter, such that the resulting allelic states are string
-    representations of the allele number.
-
-    :param int start_allele: The numeric allele to start at, subsequent alleles
-       will be created by incrementing from this number.
-
+    An *infinite alleles* model of mutation. This works by keeping track of a "next
+    allele": each time the model is asked to produce a new allele (either for an
+    ancestral or derived state), the "next allele" is provided, and then
+    incremented.
+
+    :param int start_allele: The nonnegative integer to start assigning alleles from.
+       (default: 0)
+    :ivar int next_allele: The *next* allele to be assigned. This increments by
+        one each time an allele is assigned to a new mutation, and resets to
+        ``start_allele`` each time a new ancestral state is assigned.
+    :param int block_size: TODO
```

Comment:
I don't see any reference to `block_size` in the code for this class, unless I missed something inherited?